### ISSUE TITLE
NaN on flying resources (Metal)

### ISSUE THREAD

**Issue by cmartinezledo** [eswak/chrome-ogame-ui-extension#114]
> This was my loot:
> M: 5.014.005
> C: 2.416.172
> D:  892.702

![untitled](https://user-images.githubusercontent.com/10406699/36462874-df4aad74-16a4-11e8-918a-c84f165854c1.png)
![untitled2](https://user-images.githubusercontent.com/10406699/36462938-4273932a-16a5-11e8-9261-b6b8ca60b044.png)
![untitled3](https://user-images.githubusercontent.com/10406699/36462941-44eb4864-16a5-11e8-8f3b-9fa98222c999.png)

**Comment by cmartinezledo** [comment]
This is strange... did another attack.
And now it shows correctly. But the total loot (Metal) increased to 23.321 M

**Comment by eswak** [comment]
thanks for reporting ^^

that's wierd, perhaps your have another mission in flight that could cause this bug ? Like an alliance fight ? I  think there's an issue stating that it breaks the resources in flight indicator

**Comment by cmartinezledo** [comment]
I did saw that issue with alliance missions before reporting this. 
In my case, that attack was the only mission. Then I did a new attack, and with that new loot, I could see the total loot perfectly.

I will try to reproduce it and gather more information.
What I did at that moment to try to fix it was the following:

1. Refresh
2. Clear Cache
3. Empty Cache and Hard Reload
4. Disable and enable extension

But none of them fixed this issue. I also reviewed the functions that parse that values, but I did not find anything that called my attention. 

Just to give you more details: This happened on > s118-ar.ogame.gameforge.com
